From f2ff4162b536de702e0f18f4a32850031506c204 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Sat, 9 Mar 2013 13:24:57 +0200
Subject: ARM: i.MX6: cm-fx6: separate kernel and rootfs

Make separate partitions for kernel and root filesystem on the NAND
flash.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/board-cm-fx6.c |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-cm-fx6.c b/arch/arm/mach-mx6/board-cm-fx6.c
index 70de5c8..ba1df48 100644
--- a/arch/arm/mach-mx6/board-cm-fx6.c
+++ b/arch/arm/mach-mx6/board-cm-fx6.c
@@ -215,8 +215,13 @@ static int cm_fx6_gpmi_nand_init_pads(void)
 
 static struct mtd_partition cm_fx6_nand_partitions[] = {
 	{
-		.name	= "nand",
+		.name	= "linux",
 		.offset	= 0,
+		.size	= SZ_8M,
+	},
+	{
+		.name	= "rootfs",
+		.offset	= MTDPART_OFS_APPEND,
 		.size	= MTDPART_SIZ_FULL,
 	},
 };
-- 
1.7.9.5

